packages/tools/guiEditor/src/diagram/workbench.tsx

```diff
@@ -668,7 +668,7 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         return true;
     }
 
-    onMove(evt: React.PointerEvent) {
+    onMove() {

```

Comment:
evt is used just below on line 684, so it shouldn't be removed.

Reply:
Nice catch, thank you!